exercises/armstrong-numbers/.meta/generator/armstrong_numbers_case.rb

```diff
@@ -0,0 +1,12 @@
+require 'generator/exercise_case'
+
+class ArmstrongNumbersCase < Generator::ExerciseCase
+
+  def workload
+    assert_or_refute(expected, call_armstrong)
+  end
+
+  def call_armstrong
+    "ArmstrongNumbers.include?(#{underscore(input['number'])})"
```

Comment:
I kept this function, because otherwise the single line in workload is way over 80 characters long.

Reply:
Looks great :+1: